crates/api/src/window.rs

```diff
@@ -364,7 +364,7 @@ impl Window {
     )]
     pub fn set_hl(&mut self, ns_id: u32) -> Result<()> {
```

Comment:
We should also change this:

```suggestion
    pub fn set_hl_ns(&mut self, ns_id: u32) -> Result<()> {
```

Reply:
Done, added a changelog entry since this is part of the public api